### PR TITLE
Log sampler migrate from qps to rpm

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>. NO
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/sink/sampler/RateLimitingSampler.java [hunk 1]
```diff
@@ -34,9 +34,10 @@
 @EqualsAndHashCode(of = {"qps"})
 public class RateLimitingSampler implements Sampler {
     @Getter
-    @Setter
     private volatile int qps;
```
Comment (wu-sheng) [inline oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/sink/sampler/RateLimitingSampler.java:37]: Do we still need to keep this?

Comment (wu-sheng) [reply oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/sink/sampler/RateLimitingSampler.java:37]: You should update the document too.

**Comment by wu-sheng** [comment]
And you should update LAL script, right?

**Comment by kezhenxu94** [comment]
This patch doesn’t change the public API / script, it used qps * 60 as RPM internally.

**Comment by wu-sheng** [comment]
> This patch doesn’t change the public API / script, it used qps * 60 as RPM internally.

I know. I am afraid this could be misleading. Because `30qps * 60` is different with 1800rpm. In the worth case, you could get over 500qps sampled easily in some certain cases.

**Comment by wu-sheng** [comment]
I think we should break the API in this release.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**